### PR TITLE
chore: update main page translation

### PR DESCRIPTION
**Pull request by boojack**
image

### CODE REVIEW

File: locales/zh.json [hunk 1]
```diff
@@ -15,10 +15,10 @@
     "for": "专为",
     "teams": "团队",
     "dbas": "DBA",
-    "developers": "开发者"
+    "developers": "应用开发者"
   },
   "slogan": {
-    "safer-and-faster": "更安全更高效",
+    "safer-and-faster": "安全高效",
     "mission": "专为 DBA 和研发打造的数据库变更和版本管理工具",
```
Comment (tianzhou) [inline locales/zh.json:22]:
```suggestion
    "mission": "专为 DBA 和应用开发者打造的数据库变更和版本管理工具",
```

File: locales/zh.json [hunk 2]
```diff
@@ -130,44 +130,44 @@
...
     },
     "database-as-code": {
       "title": "Database-as-Code",
-      "description": "与托管迁移脚本的 VCS 集成，在观察到新的脚本推送事件时触发迁移管道。"
+      "description": "可与版本控制系统（VCS）集成以托管 Schema 变更文件。每当新的变更文件提交后，会自动触发变更流水线。"
     },
     "streamlined-deployment": {
-      "title": "Streamlined 部署",
-      "description": "在单个 M*N 更改矩阵管道中支持多租户 (M)、多环境 (N) 模式更改。"
+      "title": "流式部署",
```
Comment (tianzhou) [inline locales/zh.json:141]:
```suggestion
      "title": "流水线部署",
```

File: locales/zh.json [hunk 3]
```diff
@@ -130,44 +130,44 @@
...
+      "description": "通过相邻版本间的比对，来展示每次迁移产生的差异。"
     },
     "anomaly-center": {
       "title": "异常中心",
       "description": "定期扫描所有托管实例和数据库，并为用户提供整体视图以查看所有异常情况。"
     },
     "drift-detection": {
       "title": "Drift 检测",
-      "description": "通过比较记录的模式和实际的模式来自动检测 schema drift，捕获任何意外的架构更改。"
+      "description": "定期扫描所有托管的实例与数据库，并为用户提供整体视图以查看所有异常对象。"
```
Comment (tianzhou) [inline locales/zh.json:158]:
```suggestion
      "description": "定期扫描所有托管的实例与数据库，并为用户提供整体视图以查看所有异常事项。"
```

File: locales/zh.json [hunk 4]
```diff
@@ -130,44 +130,44 @@
...
     "backup-and-restore": {
-      "title": "数据的备份与还原",
-      "description": "每个数据库的自动和手动备份，环境特定的备份计划策略。使用整个恢复的迁移历史链恢复到新数据库。"
+      "title": "备份与恢复",
+      "description": "支持数据库级别的定时自动、手动备份。可按开发、生产等不同环境制定差异化备份策略。"
     },
     "role-based-access-control": {
       "title": "基于角色的访问控制",
-      "description": "两个角色集。所有者、DBA、开发人员用于工作区管理。所有者、开发人员用于项目管理。"
+      "description": "提供两类角色集。Owner, DBA, Developer 用于 DBA 全局平台管理。Owner, Developer 用于应用开发者在各自项目内管理。"
```
Comment (tianzhou) [inline locales/zh.json:166]:
```suggestion
      "description": "提供两类角色集。Owner, DBA, Developer 用于全局平台管理。Owner, Developer 用于应用开发者在各自项目内管理。"
```

File: locales/zh.json [hunk 5]
```diff
@@ -212,44 +212,43 @@
     "all-rights-reserved": "版权所有。"
   },
   "index-page": {
-    "title": "{slogan}数据库变更{br}以及版本控制{br}专为{teams}而设计",
-    "description": "Bytebase 提供了一个基于 Web 的协作工作空间来帮助 {dbas} 和 {developers} 管理应用程序数据库模式 (DDL) 和数据 (DML) 的生命周期。",
+    "title": "{slogan}专为{teams}协同设计{br}数据库变更{br}以及版本控制",
+    "description": "Bytebase 是一个网页版的团队协同开发工具，帮助 {dbas} 和{developers}管理数据库 Schema (DDL) 和数据 (DML) 的生命周期。",
```
Comment (tianzhou) [inline locales/zh.json:216]:
```suggestion
    "description": "Bytebase 是一个开源数据库 DevOps 工具，帮助 {dbas} 和{developers}管理数据库 Schema (DDL) 和数据 (DML) 的生命周期。",
```

File: locales/zh.json [hunk 6]
```diff
@@ -259,45 +258,38 @@
       }
     },
     "a-complete-rethink-to-deliver-experience": {
-      "title": "彻底重新思考交付",
-      "experience": "经验",
-      "description": "从数据建模到技术堆栈、系统架构，再到最终的用户界面，Bytebase 从头开始设计，旨在为 DBA 和开发人员提供变革性的用户体验。"
+      "title": "彻底重新思考{deliver-experience}",
+      "deliver-experience": "交付体验",
+      "description": "从数据建模到技术选型，到系统架构，再到终端用户界面，Bytebase 的设计从零开始，旨在为 DBA 与开发人员提供革命性的用户体验。"
```
Comment (tianzhou) [inline locales/zh.json:263]:
```suggestion
      "description": "从数据建模到技术选型，到系统架构，再到终端用户界面，Bytebase 从头开始设计，旨在为 DBA 与开发人员提供革命性的用户体验。"
```

File: locales/zh.json [hunk 7]
```diff
@@ -259,45 +258,38 @@
...
           "title": "即插即用",
-          "description": "Bytebase 提供了一个全面的 Web 控制台，需要零配置和外部依赖。 开发人员和 DBA 可以立即专注于他们的数据库任务。"
+          "description": "Bytebase 提供了一个无需配置，零外部依赖的 Web 平台，开发者与 DBA 们可以立马投入到他们的数据库工作中。"
         },
         "usability": {
-          "title": "可用性",
-          "description": "作为管理应用程序最关键资产——事务数据库模式的专业工具，Bytebase 深思熟虑以呈现相关信息，设计适当的人体工程学。 进行模式更改总是很可怕，我们希望消除",
-          "confusing-interface": "混乱的界面"
+          "title": "易用性",
+          "description": "Bytebase 作为一款专业工具，要管理应用程序最重要的资产—— 在线数据库的 Schema。我们认真思考了如何呈现正确的信息，做出符合人体工程学的设计。执行 Schema 变更总是令人感到恐惧，我们希望消除由于{confusing-interface}而导致的风险。",
```
Comment (tianzhou) [inline locales/zh.json:275]:
```suggestion
          "description": "Bytebase 作为一款专业工具，要管理应用程序最重要的资产 —— 在线数据库的 Schema。我们认真思考了如何呈现正确的信息，做出符合人体工程学的设计。执行 Schema 变更总是令人感到恐惧，我们希望消除由于{confusing-interface}而导致的风险。",
```

File: locales/zh.json [hunk 8]
```diff
@@ -259,45 +258,38 @@
...
+          "confusing-interface": "困惑的界面"
         },
         "extensibility": {
           "title": "可扩展性",
-          "description": "作为基础设施工具包之一，Bytebase 将提供与现有 IM、CI/CD、问题跟踪、监控、登录系统集成的钩子。"
+          "description": "作为基础架构工具箱中的一员，Bytebase 提供扩展能力以便与现有的 IM 系统、CI/CD 系统、工单追踪系统、监控系统、登录系统等集成。"
```
Comment (tianzhou) [inline locales/zh.json:280]:
```suggestion
          "description": "作为基础架构工具箱中的一员，Bytebase 提供扩展能力以便与现有的 IM 系统、CI/CD 系统、工单系统、运维监控系统、登录系统等集成。"
```

File: locales/zh.json [hunk 9]
```diff
@@ -259,45 +258,38 @@
...
       }
     },
     "single-binary-with-curated-tech-stack": {
-      "title": "具有精选技术栈的单个二进制文件",
-      "the-entire-app-is-bundled-in": "整个应用程序捆绑在",
-      "a-single-binary": "单个二进制文件",
-      "and-started-via": "并通过",
-      "bytebase": "./bytebase",
-      "description": "作为专业工具，Bytebase 力求务实。 而且由于 Bytebase 是开源的，它偏爱简单的技术，并且自以为是地提供易于学习、使用和构建产品的可访问和有凝聚力的体验。 我们不希望人们浪费时间弄清楚如何部署或争论更好的编码风格。",
+      "title": "内含精选技术栈的单个编译文件",
```
Comment (tianzhou) [inline locales/zh.json:285]:
```suggestion
      "title": "精选技术栈下的单个二进制编译文件",
```

File: locales/zh.json [hunk 10]
```diff
@@ -259,45 +258,38 @@
...
+      "description": "整个应用被打包为{a-single-binary}，通过 {bytebase-link} 直接启动。作为一款开源的专业工具，Bytebase 本身追求实用性，我们偏好朴实的技术，坚持提供顺畅的用户体验，以易于用户学习、使用和改进该产品。我们不希望人们将时间浪费在解决部署问题或是争论哪种编码风格更好上面。",
```
Comment (tianzhou) [inline locales/zh.json:286]:
```suggestion
      "description": "整个应用被打包为{a-single-binary}，通过{bytebase-link}直接启动。作为一款开源的专业工具，Bytebase 本身追求实用性，我们偏好朴实的技术，坚持提供顺畅的用户体验，以易于用户学习、使用和改进该产品。我们不希望人们将时间浪费在解决部署问题或是争论哪种编码风格更好上面。",
```

File: locales/zh.json [hunk 11]
```diff
@@ -212,44 +212,43 @@
     "all-rights-reserved": "版权所有。"
   },
   "index-page": {
-    "title": "{slogan}数据库变更{br}以及版本控制{br}专为{teams}而设计",
-    "description": "Bytebase 提供了一个基于 Web 的协作工作空间来帮助 {dbas} 和 {developers} 管理应用程序数据库模式 (DDL) 和数据 (DML) 的生命周期。",
+    "title": "{slogan}专为{teams}协同设计{br}数据库变更{br}以及版本控制",
+    "description": "Bytebase 是一个网页版的团队协同开发工具，帮助 {dbas} 和{developers}管理数据库 Schema (DDL) 和数据 (DML) 的生命周期。",
     "features": {
       "database-migration": {
         "slogan": "更安全、更快速的数据库迁移 - DDL、DML",
```
Comment (tianzhou) [inline locales/zh.json:219]:
```suggestion
        "slogan": "更安全、更高效的数据库变更 - DDL、DML",
```

File: locales/zh.json [hunk 12]
```diff
@@ -212,44 +212,43 @@
...
-        "industry-first-database": "行业第一的数据库",
+        "description": "行业首个面向 DBA 和应用开发者 {collaboration} 的数据库 {devops} 解决方案",
```
Comment (tianzhou) [inline locales/zh.json:220]:
```suggestion
        "description": "行业首个面向 DBA 和应用开发者{collaboration}的数据库 {devops} 解决方案",
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by boojack** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**